src/pages/Scholarships.jsx

```diff
@@ -20,23 +20,26 @@ function ScholarshipsPage() {
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
+  const [sortBy, setSortBy] = useState('deadline');
+  const [sortFormat, setSortFormat] = useState('asc');
 
   useEffect(() => {
     // TODO: Create cancellable promises
-    Scholarships.list({ sortField: 'deadline' })
+    Scholarships.list({ sortField: `${sortBy}`, sortDir: `${sortFormat}` })
```

Comment:
`stringVar` is the same as `${stringVar}`.

Also, I suggest renaming the new state variables to `sortField` and `sortDir` so you can simply do this instead:
```jsx
    Scholarships.list({ sortField, sortDir })
```

Reply:
ok